pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -365,8 +368,9 @@ def solve(self, model, **kwds):
             #     MindtPy.feas_inverse_map[n] = c
 
             # Create slack variables for OA cuts
-            lin.slack_vars = VarList(
-                bounds=(0, config.max_slack), initialize=0, domain=NonNegativeReals)
+            if config.add_slack:
+                lin.slack_vars = VarList(
+                    bounds=(0, config.max_slack), initialize=0, domain=NonNegativeReals)
```

Comment:
Change not required for approval:

This isn't necessary, since you just end up with a dimension 0 singleton VarList.

Reply:
We decided to you remove the variables since it pollutes the master problem with an empty variable definition